packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -106,7 +106,7 @@ class WebFlutterDriver extends FlutterDriver {
       //
       // TODO(angjieli): Support Mobile Web. (https://github.com/flutter/flutter/issues/65192)
       await webDriver.execute(
-          "document.querySelector('flt-semantics-placeholder').click();",
+          '(document.querySelector("flt-glass-pane")?.shadowRoot ?? document).querySelector("flt-semantics-placeholder").click();',
```

Comment:
This needs to be reverted after this lands: https://github.com/flutter/flutter/pull/82939

Reply:
Reverted